play/tests/unittests/PhaseResults.test.tsx

```diff
@@ -70,22 +70,25 @@ export function renderWithThemeRouterTranslation(
 
 // function for getting answer choices from a question
 const getAnswerChoices = (mockCurrentQuestion: IQuestion) => {
-  return mockCurrentQuestion?.choices?.map((choice: IChoice) => ({
+  return (
+    mockCurrentQuestion?.choices?.map((choice: IChoice) => ({
       id: uuidv4(),
       text: choice.text,
       isCorrectAnswer: choice.isAnswer,
       reason: choice.reason ?? '',
-  })) ?? [];
+    })) ?? []
+  );
 };
 
-describe ('PhaseResults', () => {
+describe('PhaseResults', () => {
   // tests if player has answered incorrectly on phase 1 (starting score: 0, ending score: 0)
   it('Phase 1, wrong answer', async () => {
```

Comment:
Similar to other comment, for all `it('XXX` in this file, probably better to rephrase them so they can be a full English sentence and `it` plays the subject in that sentence. Also a bit details on what it is testing like: `it('doesn't increase score when wrong answer is selected in Phase 1` Basically this should be used to explain the details about the test so you can omit commenting above the test

Reply:
corrected